picard/tagger.py

```diff
@@ -416,7 +426,11 @@ def process(result=None, error=None):
             if result:
                 if error is None:
                     self.add_files(result)
-                thread.run_task(get_files, process)
+                thread.run_task(
+                    get_files,
+                    process,
+                    priority=2,
+                    thread_pool=self.load_thread_pool)
```

Comment:
We run this on the load threadpool to avoid it blocking, and run it at priority 2 so that it runs ahead of queued tasks to load metadata from files.

But I wonder whether it might make more sense to have another threadpool just for this.